features/apply.feature

```diff
@@ -150,6 +150,11 @@ Feature: Apply command
     Then the output should contain "Could not find stack definition bar in region foo"
     And the exit status should be 1
 
+  Scenario: Run apply with stack in wrong region
+    When I run `stack_master apply foo myapp-web`
+    Then the output should contain "Stack name myapp-web exists in regions: us-east-1"
+    And the exit status should be 1
+
```

Comment:
This is interesting, I opted for unit tests rather than features.  I wonder which is better 🤔

Reply:
This level of testing can also be done via rspec+aruba. I opted for that approach in the drift feature and I'm wondering if we should re-write all the cucumber features to use rspec instead. https://github.com/envato/stack_master/pull/339/files#diff-0845fa726c236bab4b7365489a3cd108R1

Reply:
In this case there was an existing cucumber test for the output of this command, so it was the easiest solution.

Reply:
Yep, 👍. Sorry, I wasn't suggesting that this change needed to move to rspec.